src/styles/block-card.css

```diff
@@ -0,0 +1,97 @@
+/* #region Card */
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  color: #060b35;
+  --button-color: #00acdc;
+  --button-height: 40px;
+}
+
+.card__box-img {
+  display: flex;
+  padding: 0 3px;
+  margin: 0 0 40px;
+}
+
+.card__img {
+  width: 100%;
+  height: 134px;
+}
+
+.card__name {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__code,
+.card__reviews,
+.card__text-price {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  columns: #616070;
+  margin: 0 0 16px;
+}
+
+.card__evaluation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 16px;
+}
+
+.card__text-price {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
```

Comment:
You can also declare var with this color, because you use thi color in different places

Reply:
in my opinion, these colors may to do different